archlinuxcn/blackbox-terminal/lilac.yaml

```diff
@@ -0,0 +1,18 @@
+maintainers:
+  - github: wszqkzqk
+    email: wszqkzqk <wszqkzqk@qq.com>
+
+build_prefix: extra-x86_64
+
+pre_build_script: |
+    update_pkgver_and_pkgrel(_G.newver)
+
+post_build_script: |
+    git_pkgbuild_commit()
+
+update_on:
+  - source: gitlab
+    gitlab: raggesilver/blackbox
+    host: gitlab.gnome.org
+    use_latest_release: true
```

Comment:
There is no `use_latest_release` config for gitlab, check also the [doc](https://nvchecker.readthedocs.io/en/latest/usage.html#check-gitlab) of nvchecker.